dot/rpc/modules/childstate_test.go

```diff
@@ -276,6 +276,6 @@ func setupChildStateStorage(t *testing.T) (*ChildStateModule, common.Hash) {
 	err = st.Block.AddBlock(b)
 	require.NoError(t, err)
 
-	hash, _ := st.Block.GetBlockHash(b.Header.Number)
+	hash, _ := st.Block.GetHashByNumber(b.Header.Number)
```

Comment:
I see that previously also we were ignoring this error, but we should check it.

Reply:
thanks, done in d9ac108